README.md

```diff
@@ -22,6 +22,9 @@ To run the docs web application locally you should do the following:
 git clone https://github.com/resin-io/docs
 cd docs
 npm install
+npm install -g bower
```

Comment:
NIT, but I prefer using the bundled version (in which case the path should be adjusted, of course).